lib/customServiceAPI.js

```diff
@@ -176,7 +176,32 @@ class customServiceAPI  {
 		}
 
 	}
+    
+	/**
+	 * Sends a read request to the KNX bus. Answer will returned by a call of onKNXValueChange.
+	 * 
+	 * @param {string} field - The name of the characteristic. Requires a "Listen" in the characteristic or KNXObject. 
+	 */
+	knxReadRequest (field) {
+		/** @type {CharacteristicKNX} */
+		var chrKNX;
+		this.serviceKNX.globs.info(this.handlerName + "->customServiceAPI.knxReadRequest(" + field + ")");
+
+		// get the characteristic
+		// iterate(this.characteristicsList);
+		if (!this.characteristicsList[field]) {
+			throw (new Error('HANDLER CONFIGURATION ERROR').message='Error in ' + this.handlerName + '. Field '+ field + ' does not exist');
+		} else {
+			chrKNX = this.characteristicsList[field];
+		}
+		// iterate through all group addresses to be written for that characteristic
+		for (var iGA = 0; iGA < chrKNX.setGroupAddressList.length; iGA++) {
+			var gaddress = chrKNX.listenGroupAddressList[iGA];
```

Comment:
`chrKNX.setGroupAddressList` != `chrKNX.listenGroupAddressList`
I think you are meaning the **chrKNX.listenGroupAddressList** in both cases?